client/softlayer_client.go

```diff
@@ -169,6 +178,15 @@ func (slc *SoftLayerClient) GetSoftLayer_Hardware_Service() (softlayer.SoftLayer
 	return slService.(softlayer.SoftLayer_Hardware_Service), nil
 }
 
+func (slc *SoftLayerClient) GetSoftLayer_Dns_Domain_Resource_Record_Service() (softlayer.SoftLayer_Dns_Domain_Resource_Record_Service, error) {
+	slService, err := slc.GetService("SoftLayer_Dns_Domain_ResourceRecord")
```

Comment:
This is most likely an oversight on our part, I'm in agreement with matching the SL name as it creates consistency. I'll make sure this is addressed in a PR.